uwblueprint/jump-math
# fix: enable delete for active sessions

## Notion ticket link

[Enable deleting active test sessions](https://www.notion.so/uwblueprintexecs/Enable-deleting-active-test-sessions-a6a11c5dd81a48899824cbdf03c91989?pvs=4)



## Implementation description
* Add option to delete active test sessions in popover UI
* Update backend function and unit tests to support active sessions
* Delete `EmptyStudentsMessage.tsx` (unused)



## Steps to test
1. Create an upcoming and active test session. 
2. Verify that both test sessions you created have the delete option.
3. Verify that deleting is successful.
4. Verify that past test sessions do not have the delete option (and no popover in general)



## What should reviewers focus on?
* 


## Checklist
